docker-compose.yml

```diff
@@ -70,7 +70,8 @@ services:
     image: node:14-alpine
     environment:
       APPEALS_SERVICE_API_URL: http://appeals-service-api:3000
-      DOCUMENT_SERVICE_API_URL: http://document-service-api:3000
+#      DOCUMENTS_SERVICE_API_URL: http://document-service-api:3000
+      DOCUMENTS_SERVICE_API_URL: https://virtserver.swaggerhub.com/riggerthegeek/docs-service-api/1.0.0
```

Comment:
This is a transient one - use the local host

Reply:
Changed as suggested - this breaks the e2e as `http://document-service-api:3000` isn't returning an `id` as of yet. 

Not sure how to proceed - rest of the PR is good for review.

Reply:
Locally I have the impression that it's still in asynchronous mode. We are not receiving any apiResponse.

I tried with 
https://virtserver.swaggerhub.com/riggerthegeek/docs-service-api/1.0.0
and it's fine